database/resource.go

```diff
@@ -68,6 +71,10 @@ type Resource struct {
 	// required: true
 	// example: address
 	Address string `json:"address"`
+    // Default state of the Resource
+    // required: true
+    // example: true
+    IsDefault bool `json:"is_default"`
```

Comment:
Would it be alright to change `is_default` to `isDefault`? All the data model properties in the UI are camelCase.

Reply:
sure thing